OfficeDev/office-ui-fabric-react
# Scaffold @uifabric/lists package via create-package script

#### Description of changes

The intent of the `@uifabric/lists` package is to prototype various approaches to lists both virtualized and non-virtualized.

Eventually, this will be a home for _all_ Fabric List controls, migrated in a backwards-compatible way.

#### Focus areas to test

1. Run `cd .\packages\lists\`
1. Run `npm start`
1. Bootstrapped component Fabric local dev site should render

cc: @soettl @dmichon-msft @ThomasMichon 

Related to #5557.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9106)
